.eslintrc.yaml

```diff
@@ -85,7 +85,7 @@ rules:
   implicit-arrow-linebreak: [warn, beside]
   indent: [warn, 2, {SwitchCase: 1}]
   jsx-quotes: [warn, prefer-double]
-  key-spacing: [warn, {beforeColon: false, afterColon: true}]
+  key-spacing: [warn, {align: value}]
```

Comment:
Does it require keys to be aligned everywhere, or just allow it? I wouldn't very much like enforcing alignment everywhere, since you often have to adjust the spacing for an entire object when you add/remove a key, and that increases diff size/noise for not much readability gain (though it's nice for large objects like `TYPE_OPTIONS`).

Reply:
Agreed, it shouldn’t enforce it. I relaxed `key-spacing` rule with [minimum mode](https://eslint.org/docs/rules/key-spacing#mode) in https://github.com/metabrainz/musicbrainz-server/pull/639/commits/847229904fb91677849eb0906b853a231551e4a7.